pkg/services/object/acl/v2/service.go

```diff
@@ -22,11 +24,26 @@ import (
 	"go.uber.org/zap"
 )
 
+type sessionTokenCheckResultKey struct {
+	tokenChecksum [sha256.Size]byte
+	// request properties for which session token validation is static
+	requestVerb        sessionSDK.ObjectVerb
+	requestedContainer cid.ID
+	requestedObject    oid.ID
```

Comment:
This makes the cache less useful. Most of the time it's wildcard wrt OIDs, this should be leveraged.